.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "node"
+
+branches:
+  only:
+    - master
+
+jobs:
+  include:
+    - stage: "Lint" # naming the Tests stage
+      script: npm run lint
```

Comment:
By default `npm run lint` will run `vue-cli-service lint` which will try to automatically fix errors (https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint#injected-commands). Add a `--no-fix` in the `lint` command to avoid auto-fixing from CI.

Reply:
Thanks! I made those adjustments.